Makefile

```diff
@@ -255,8 +251,8 @@ $(EMBEDMD_BINARY):
 $(JB_BINARY):
 	@go install -mod=vendor github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
 
-$(PO_CRDGEN_BINARY): cmd/po-crdgen/main.go $(OPENAPI_TARGET)
-	@go install -mod=vendor $(GO_PKG)/cmd/po-crdgen
+$(CONTROLLER_GEN_BINARY):
+	@go install -mod=vendor sigs.k8s.io/controller-tools/cmd/controller-gen 
```

Comment:
don't use an arbitrary version, but download a binary. Controller tools is often lagging behind in Go and Kube version support. I would try to avoid any trouble by using a binary. We use those from https://github.com/openshift/kubernetes-sigs-controller-tools/releases.

Reply:
I was just following what we are doing for the other build tools here which AFAIK is preferred since it avoids committing binary tools to git or downloading them at build time.  It's not an arbitrary version because it's installed from the vendor directory set to v0.2.2.  But I can certainly change it if that's preferred, @s-urbaniak wdyt?

Reply:
@pgier I am voting for pinning the version in the Makefile :+1: